manifests/k/K1JT/wsjtx/v2.6.1/K1JT.wsjtx.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: K1JT.wsjtx
+PackageVersion: v2.6.1

```

```suggestion
PackageVersion: 2.6.1
```
This should match control panel. After changing this value, the name of the folder will need to be updated.